### PR TITLE
Sorting myscans and starredscans by DESC

### PR DESCRIPTION
**Pull request by DynaMc**
Sorting myscans and starredscans table ID column to DESC. So that the latest scan shows up at the top of the list. It really helps when looking at myscans when you have submitted many links.

### CODE REVIEW

File: templates/interface/myscans2.html [hunk 1]
```diff
@@ -254,9 +256,10 @@ <h3 class="panel-title">Starred Scans</h3>
             b = parseInt($(b).text());
             return ((a < b) ? 1 : ((a > b) ?  -1 : 0));
         };
-
+*/
```
Comment (Dennisparchkov) [inline templates/interface/myscans2.html]: Yes, new scans at the top of the list is a good idea. 
Would it be possible to remove  `/*` and `*/` in code, i don't see their purpose (or i might be wrong, explain why you used them) ?

Comment (DynaMc) [reply templates/interface/myscans2.html]: I was commenting out the old code but it can be removed.

**Comment by Dennisparchkov** [comment]
I've looked into to it and think you should also remove 

``` python
"aoColumns" : [
                    { "sType" : "numeric-html" },
                    null,
                    null
                ]
```

from both DataTables() initializations.  I'm getting an error because it refers to the part you commented out (numeric-html).
If I remember correctly i had some issues sorting scans by ID number because it was not recognized as a digit but as a string. The commented out part created the sorting (descending and ascending) from the string. But it seems to work without it by descending order.

**Comment by DynaMc** [comment]
It does not appear to be working as well as I'd though when I first tried. see below.

image

**Comment by DynaMc** [comment]
Nope this is not the solution. Tested it more and the table goes all crazy. Will re-submit if I happen to find a way to do it.

**Comment by Dennisparchkov** [comment]
That is the exact same problem I described in my previous comment and was having when adding DataTables. The field is not being recognized as a digit. 
What i suggest you do is keep the parts you removed (previous 2 commits) as this is the code to sort the ID field by ascending and descending order.

``` python
jQuery.fn.dataTableExt.oSort['numeric-html-asc']  = function(a, b) {        
             a = parseInt($(a).text());     
             b = parseInt($(b).text());     
             return ((a < b) ? -1 : ((a > b) ?  1 : 0));        
         };     

         jQuery.fn.dataTableExt.oSort['numeric-html-desc']  = function(a, b) {      
             a = parseInt($(a).text());     
             b = parseInt($(b).text());     
             return ((a < b) ? 1 : ((a > b) ?  -1 : 0));        
         };
```

and 

``` python
"aoColumns" : [
                    { "sType" : "numeric-html" },
                    null,
                    null
                ]
```

Keep your default sort by descending order line

``` python
"aaSorting": [[0, "desc"]],
```

**Pull request closed without merging**